src/main/scala/it/pagopa/interop/catalogmanagement/server/impl/Dependencies.scala

```diff
@@ -54,8 +54,8 @@ trait Dependencies {
 
   val uuidSupplier: UUIDSupplier = new UUIDSupplierImpl
 
-  def getFileManager(): Future[FileManager] =
-    FileManager.getConcreteImplementation(StorageConfiguration.runtimeFileManager).toFuture
+  def getFileManager(blockingEc: ExecutionContextExecutor): FileManager =
+    FileManager.get(FileManager.S3)(blockingEc)
```

Comment:
I'll love to add PureConfig before this change to don't have to explicitly map `"S3"` to `FileManager.S3`. BTW This PR was meant to solve compilation issues deriving from the new library shape. Do you mind if I track this request in a Refactor PR to handle it in a second moment?

Reply:
I'm asking it just to be able to make the project recompile and to focus on the job deploy

Reply:
I see, my only concern is that adding PureConfig could take a bit longer (because a bit invasive).

If you agree, I would prefer to keep the existing features with a temporary old-fashioned reading from configs and improve it with the PureConfig task, than loosing the functionality and having it back later.